common/lib/xmodule/xmodule/templates/problem/circuitschematic.yaml

```diff
@@ -3,9 +3,9 @@ metadata:
     display_name: Circuit Schematic Builder
     markdown: !!null
 data: |
-      <problem >
-        Please make a voltage divider that splits the provided voltage evenly.
-    
+      <problem>
+        <fieldset>
+          <legend>Make a voltage divider that splits the provided voltage evenly.</legend>
```

Comment:
unlike the others of these problem types, there's no introductory statement for this one about what this problem type is or how to use it.